core/domain/event_services.py

```diff
@@ -64,18 +79,25 @@ class StatsEventsHandler(BaseEventHandler):
     stats data.
     """
 
-    EVENT_TYPE = feconf.EVENT_TYPE_ALL_STATS
+    EVENT_TYPE: str = feconf.EVENT_TYPE_ALL_STATS
 
     @classmethod
-    def _is_latest_version(cls, exp_id, exp_version):
+    def _is_latest_version(cls, exp_id: str, exp_version: int) -> bool:
         """Verifies whether the exploration version for the stats to be stored
         corresponds to the latest version of the exploration.
         """
         exploration = exp_fetchers.get_exploration_by_id(exp_id)
-        return exploration.version == exp_version
+        return bool(exploration.version == exp_version)
```

Comment:
Why we need to do `bool()`?

Reply:
actually `exploration.version` belongs to exp_domain and that file is not annotated in this PR, but now bool is removed by adding only necessary changes in exp_domain.

Reply:
Thx!